src/components/Login/index.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from "react";
+import { Form, Input, Button, Checkbox, Alert } from "antd";
+import { useHistory } from "react-router-dom";
+import firebase from "../../firebaseConfig";
+import { MailOutlined } from "@ant-design/icons";
+import { Link } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+
+const layout = {
+  labelCol: { span: 4 },
+  wrapperCol: { span: 8 },
+};
+
+const tailLayout = {
+  wrapperCol: { offset: 8, span: 16 },
+};
+
+const SignUp = ({ isTesting }) => {
+  const { t } = useTranslation();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const login = (e) => {
```

Comment:
and there's a login function in side the signup component

Reply:
ı changed the function name as signin